react/features/overlay/components/OverlayContainer.js

```diff
@@ -95,6 +121,25 @@ class OverlayContainer extends Component {
     }
 
     /**
+     * Initializes a new ReloadTimer instance.
+     *
+     * @param {Object} props - The read-only properties with which the new
+     * instance is to be initialized.
+     * @public
+     */
+    constructor(props) {
+        super(props);
+        this.state = {
+            /**
+             * Indicates whether the film strip only mode is enabled or not.
+             *
+             * @type {boolean}
+             */
+            filmStripOnly: interfaceConfig.filmStripOnly
```

Comment:
interfaceConfig will go into the Redux store. Then it will be fetched from there into a Component prop. Once something's in the Redux store, it's no worth limiting ourselves to thinking of it as constant. A React Component represents a function over its props and state, so we're setting ourselves now to be prepared to re-render on props or state changes. I've talked about this with @hristoterezov before.